### PR TITLE
refactor: ziofy PermissionsResponderADM  DEV-1739

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

Issue Number: DEV-1739

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
LGTM

File: webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala [hunk 1]
```diff
@@ -27,30 +29,57 @@ import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectADM
...
+   */
+  def userAdministrativePermissionsGetADM(
+    groupsPerProject: _root_.scala.collection.immutable.Map[
+      _root_.org.knora.webapi.IRI,
+      _root_.scala.collection.immutable.Seq[_root_.org.knora.webapi.IRI]
+    ]
+  ): zio.Task[_root_.scala.collection.immutable.Map[_root_.org.knora.webapi.IRI, _root_.scala.collection.immutable.Set[
+    _root_.org.knora.webapi.messages.admin.responder.permissionsmessages.PermissionADM
+  ]]]
+}
```
[the remark this answers is not in the document]
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala:62]: > The special predeﬁned name _root_ refers to the outermost root package which contains all top-level packages.
http://www.scala-lang.org/docu/files/ScalaReference.pdf

[the remark this answers is not in the document]
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala]: fixed in 93d5a0e60c8864aba2cf6bfe472790bbe4695f0a

**Review by irinaschubert — APPROVED** [review]
LGTM

**Review by mpro7 — APPROVED** [review]

**Pull request merged**